kogito-integration-tests/kogito-quarkus-integration-test/pom.xml

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-universe-kogito-integration-tests-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>kogito-quarkus-integration-test</artifactId>
+
+    <properties>
+        <kogito.version>0.8.0</kogito.version>
```

Comment:
I was thinking of `quarkus-universe-parent` together with the Camel Quarkus version.

Reply:
Hmmm, and indeed you added it in the main parent pom. Is there a reason it's duplicated here?

Reply:
@gsmet The problem here is that I have to add those dependencies explicitly (together with their version) because otherwise the quarkus bom 1.3.0 will bring in kogito 0.7.1 and this will break the tests. This problem will be solved with quarkus 1.4 which won't contain kogito anymore.